client/src/services/eventType/Normalizer.js

```diff
@@ -0,0 +1,25 @@
+export const EventTypeMapper = EventType => ({
```

Comment:
call service **files** in camel case, starts from small letter. And move this normalizer to eventType **module**, as it relates for that data